Cargo.toml

```diff
@@ -4,17 +4,26 @@ version = "0.1.10"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/prisma/connection-string"
 documentation = "https://docs.rs/connection-string"
-description = "Connection string parsing in Rust"
+description = "Connection string parsing in Rust (and WebAssembly)"
 readme = "README.md"
 edition = "2018"
 keywords = []
 categories = []
 authors = [
-    "Yoshua Wuyts <yoshuawuyts@gmail.com>"
+    "Yoshua Wuyts <yoshuawuyts@gmail.com>",
+    "Julius de Bruijn <julius@nauk.io>",
 ]
 
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = ["-Oz", "--enable-mutable-globals"]
+
 [features]
+wasm = ["wasm-bindgen"]
 
 [dependencies]
+wasm-bindgen = { version = "0.2", optional = true }
```

Comment:
I'm not super fresh on compiling to wasm, but couldn't we scope by target instead of feature here? (so no wasm feature, but conditionally include wasm-bindgen only on the wasm32-unknown-unknown target)

Reply:
https://github.com/prisma/connection-string/pull/6/commits/866e443c88015c023ff471e5c7c93271ba97f973